tests/integration/delete.bats

```diff
@@ -276,3 +276,52 @@ EOF
 	# Expect "no such unit" exit code.
 	run -4 systemctl status $user "$SD_UNIT_NAME"
 }
+
+@test "runc delete after create process killed" {
+	# This test verifies that a container can be properly deleted
+	# even if the runc create process was killed with SIGKILL
+
+	[ $EUID -ne 0 ] && requires rootless_cgroup root cgroups_v1
+	set_cgroups_path
+
+	# Add resource limits to slow down cgroup creation
+	update_config '.linux.resources.memory.limit = 67108864' # 64MB
+	update_config '.linux.resources.cpu.shares = 100'
+	update_config '.linux.resources.cpu.quota = 10000'
+	update_config '.linux.resources.cpu.period = 100000'
+
+	# Add many device rules to further slow down cgroup creation
+	update_config '.linux.resources.devices = []'
+	for i in {1..300}; do
+		update_config '.linux.resources.devices += [{"allow": true, "access": "rwm", "type": "c", "major": '"$i"', "minor": 0}]'
+	done
+
+	# Start runc create and kill it after 5ms with SIGKILL
+	timeout --signal=SIGKILL --kill-after=0 0.05s "$RUNC" --debug ${RUNC_USE_SYSTEMD+--systemd-cgroup} --root "$ROOT/state" create --console-socket "$CONSOLE_SOCKET" test_create_killed || true
+
+	# Wait briefly to ensure background processes complete
+	sleep 1
+
+	# Check container state - should be in stopped state after SIGKILL
+	runc state test_create_killed
+	[ "$status" -eq 0 ]
+	[[ "$output" == *"stopped"* || "$output" == *"paused"* ]]
```

Comment:
If the container should be in stopped state, why you also check for paused? Either the comment or the code is wrong.